finance-api/overview.md

```diff
@@ -0,0 +1,18 @@
+Finance API overview
+===
+Merchants taking payments with Zettle have internal accounts that handle transactions for the payments. For example, card payments made with Zettle card readers.
```

Comment:
I found this part bit hard to understand. This feels a bit confusing to me. 
In my opinion, I think it's best to explain things simple as possible in simple English.
Not sure what is the best way to phrase this, maybe something like this

"The Finance API is used to fetch finance related information processed through Zettle this includes account balance, all the transactions process through Zettle and payout information"

What do you think?